### PR TITLE
fix(572): arm docker build

### LINKED ISSUE #572: Fix arm64 docker build
The problem in building `arm64` image:
```
#18 [linux/arm64 builder 6/7] RUN cargo build --release
#18 5.308     Updating crates.io index
#18 68.44 Killed
#18 ERROR: process "/bin/sh -c cargo build --release" did not complete successfully: exit code: 137
```

Build log: https://github.com/dotenv-linter/dotenv-linter/actions/runs/3264295030/jobs/5364708618

Related issues:
* https://github.com/docker/buildx/issues/359
* https://github.com/docker/buildx/issues/644

### PR DESCRIPTION
**Pull request by DDtKey**
Closes #572

It's quite common issue and there are a lot of issues around GH regarding this (e.g: https://github.com/docker/build-push-action/issues/621, https://github.com/rust-lang/cargo/issues/10583)
I guess it should help in our case as well

#### ✔ Checklist:

- [X] Commit messages have been written in [Conventional Commits](https://www.conventionalcommits.org) format;
- [ ] Tests for the changes have been added (for bug fixes / features);
- [ ] Docs have been added / updated on the [dotenv-linter.github.io](https://github.com/dotenv-linter/dotenv-linter.github.io) (for bug fixes / features).

### CODE REVIEW

**Comment by DDtKey** [comment]
@mgrachev hey, looks working

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
 
+WORKDIR /
+
 RUN cargo new dotenv-linter
 COPY Cargo.toml ./
-COPY src ./src
-COPY benches ./benches
+COPY src src
+COPY benches benches
+COPY .cargo .cargo
```
Comment (mgrachev) [inline Dockerfile:9]:
```suggestion
ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
```
Maybe we will just use the environment variable instead of adding a file?

Comment (DDtKey) [reply Dockerfile:9]: I believe it's more extendable way, we probably would need to add other configs in the future

**Comment by mgrachev** [comment]
@DDtKey Thanks a lot 🙂

**Pull request merged**